src/components/TextInput/BaseTextInput.js

```diff
@@ -285,7 +285,7 @@ class BaseTextInput extends Component {
                                             (!hasLabel || this.props.multiline) && styles.pv0,
                                             this.props.prefixCharacter && StyleUtils.getPaddingLeft(this.state.prefixWidth + styles.pl1.paddingLeft),
                                             this.props.secureTextEntry && styles.secureInput,
-                                            !this.props.multiline && {height: this.state.height},
+                                            !this.props.multiline && {height: this.state.height, lineHeight: null},
```

Comment:
```suggestion
                                            !this.props.multiline && {height: this.state.height, lineHeight: undefined},
```

I see that we use `undefined` instead of null for style value.